### ISSUE TITLE
Adjust toolbar classes

### ISSUE THREAD

**Issue by dshorthouse** [SpeciesFileGroup/hyclas#1]
Awesome, simple first pass at this. One suggestion is to adjust classes of buttons for :hover and :focus so it's evident to the user which button has been selected.

**Comment by mjy** [comment]
Ultimately this library won't include buttons, just the events/hooks to change type.  Users will implement the type selection however tthey want.

**Comment by dshorthouse** [comment]
> Ultimately this library won't include buttons, just the events/hooks to change type. Users will implement the type selection however tthey want.

Fair enough but is there any harm in doing both as a first pass? Injecting buttons as part of the library will reduce some of the load off developer for config & set-up, no?

**Comment by jlpereira** [comment]
Menu and library are separate classes, both could be added to the package to be used together or just the core. For now its a experiment, but its a good point. I think we can add a default one
